exercise-src/react-native/18-maps/01-solution/.env.example

```diff
@@ -0,0 +1,4 @@
+PMO_API=http://localhost:7070
+PMO_ASSETS=https://www.place-my-order.com/
+GOOGLE_OAUTH_CLIENT_ID=
+GOOGLE_OAUTH_CLIENT_ID=
```

Comment:
This is duplicated.

Reply:
I copy-pasted the wrong one. was supposed to be `GOOGLE_MAPS_API_KEY`. I've fixed it locally and it'll be in the next one tomorrow.